template/partial/settings.html

```diff
@@ -0,0 +1,49 @@
+<form class="ui form signup-form">
+	<div class="two fields">
+		<div class="field">
+			<label>Email</label>
+			<input type="text" name="email" id="email" placeholder="Email">
```

Comment:
You should not keep the value of name and id as same. Probably use, emailId for id, similarly nameId in line number 10.